src/lib/addFilesAndCreateTag.js

```diff
@@ -3,15 +3,21 @@ const terminateProcess = require('./helpers').terminateProcess;
 const shell = require('shelljs');
 const log = require('./log');
 
-module.exports = function addFilesAndCreateTag(newVersion, mockPush) {
+module.exports = function addFilesAndCreateTag(newVersion, mockPush, ci) {
   let code;
   // ###### Add edited files to git #####
   log.info('>>> About to add and commit package.json and CHANGELOG...');
   code = shell.exec('git add package.json CHANGELOG.md').code;
   terminateProcess(code);
 
   // ###### Commit files #####
-  code = shell.exec('git commit -m "chore(release): ' + newVersion + ' [ci skip] ***NO_CI***"').code;
+  let commitMessage = 'git commit -m "chore(release): ' + newVersion;
+  if (!ci) {
```

Comment:
Hi @Biboba Thanks for the PR first of all. I have a few questions here:

* `semantic-release` has option `ci` default set to `true` - it means it only adds `[ci skip]` if we set it to `--ci`.  In your case you have the opposite condition here. 
* I also find very confusing the short name `ci`. I know we agreed to this variable name but what do you think if we use `ci-skip` or `no-ci`? Second mirrors semantic-release api. 
* could we make it default `true`, like semantic-release. If so, we need to make it a `Breaking Change` in this version and analyze the impact of the new version. 
* Would you mind updating the README with the new version? I know it's a duplication of the `program.options`. I am open for suggestions here too to have only one place where we modify the options and keep readme updated. 

refs:
https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/configuration.md#ci